src/Rheologies/elasto_visco_plastic_rheology.jl

```diff
@@ -72,21 +79,29 @@ Keyword Arguments
            transitioning the ice from a plastic to a viscous behaviour. Default value is `1e-10`.
 - `min_relaxation_parameter`: Minimum value for the relaxation parameter `α`. Default value is `30`.
```

Comment:
```suggestion
- `min_relaxation_parameter`: Minimum value for the relaxation parameter `α`. Default: `30`.
```

I thought this was how we formatted the default, but we can change formatting if we want to say "default value"...